src/main/java/org/example/persistence/ormanager/ORManagerImpl.java

```diff
@@ -28,4 +28,22 @@ static DataSource createDataSource(String url, String user, String password, Map
         System.out.println(connection.isValid(1000));
         return dataSource;
     }
+    @Override
+    public Student save(Student student){
+        Connection connection;
+        try {
+           connection = dataSource.getConnection();
+            Statement statement = connection.createStatement();
+            statement.execute("CREATE TABLE IF NOT EXISTS STUDENTS (first_name VARCHAR(255)) ;");
+            System.out.println("table created");
+           PreparedStatement ps =  connection.prepareStatement("INSERT INTO STUDENTS (first_name) values(?)");
+           ps.setString(1, student.getFirstName());
+           int rows = ps.executeUpdate();
+            System.out.println(rows + " rows affected.");
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
Consider handling the exception, not throwing a new one.
Here's a good information about handling that you can use: https://docs.oracle.com/javase/tutorial/jdbc/basics/sqlexception.html